include/swift/AST/DiagnosticsSema.def

```diff
@@ -1933,6 +1933,10 @@ NOTE(objc_generic_extension_using_type_parameter_here,none,
      "generic parameter used here", ())
 NOTE(objc_generic_extension_using_type_parameter_try_objc,none,
      "add '@objc' to allow uses of 'self' within the function body", ())
+ERROR(unsupported_existential_extension,none,
+      "extension of type %0 is not supported", (Type))
+NOTE(unsupported_existential_extension_rewrite,none,
+     "did you mean to extend %0 instead?", (Type))
```

Comment:
We already have this diagnostic (see `invalid_nominal_extension_rewrite`) so I think you can just reuse it instead of adding a new one.

Reply:
Ah, of course, good point. Will do.